src/components/Comments/ReplyCommentItem.js

```diff
@@ -230,7 +314,7 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for displaying the total vote count has been updated. This change is fine as long as the `totalVote` variable is always defined and is a number.



<!-- This is an auto-generated comment by OSS CodeRabbit -->